sentences/en/weather_HassGetWeather.yaml

```
@@ -0,0 +1,10 @@
+language: en
+intents:
+  HassGetWeather:
+    data:
+      - sentences:
+          - "<what_is>[ the] weather[ like][( for| in| at) <name>]"
+          - "<what_is>[ the] weather[( for| in| at) <name>][ like]"
```

You can't have only permutations of all optional components, as missing them all would make the 2 sentences identical and, thus, ambiguous for the recognizer.
```suggestion
          - "<what_is>[ the] weather[( for| in| at) <name>] like"
```

---

I mean, you can, but you also said you shouldn't 😅

---

Ah, good point :smile: